src/lsp/cobol_data/data_picture.ml

```diff
@@ -247,7 +270,8 @@ module Symbol = struct type t = symbol let compare = Stdlib.compare end
 module Symbols = Set.Make (Symbol)
 module SymbolsMap = Map.Make (Symbol)
 
-let pp_category = TYPES.pp_category
+let pp_category = TYPES.pp_category ~show_edition:true
```

Comment:
Let's use `pp_category` and `pp_detailed_category` instead. The `show_edition` argument for `TYPES.pp_category` may just not be optional (and should be named `~with_editions` instead — so it reads English ;-) ).